$OpenBSD$

Disable ABSL_HAVE_ELF_MEM_IMAGE.

This will exclude the files
    absl/debugging/internal/self_mem_image.(h|cc)
    absl/debugging/internal/vdso_support.(h|cc)
from the build. Since OpenBSD has no Linux vdso(7), this is OK.

Index: absl/debugging/internal/elf_mem_image.h
--- absl/debugging/internal/elf_mem_image.h.orig
+++ absl/debugging/internal/elf_mem_image.h
@@ -31,7 +31,7 @@
 #error ABSL_HAVE_ELF_MEM_IMAGE cannot be directly set
 #endif
 
-#if defined(__ELF__) && !defined(__native_client__) && !defined(__asmjs__) && \
+#if defined(__ELF__) && !defined(__OpenBSD__) && !defined(__native_client__) && !defined(__asmjs__) && \
     !defined(__wasm__)
 #define ABSL_HAVE_ELF_MEM_IMAGE 1
 #endif
